tools/clang/test/HLSL/out-param-diagnostics.hlsl

```diff
@@ -170,6 +170,20 @@ void MaybeUsedMaybeUnused([maybe_unused] out int Val, int Cnt) { // expected-not
     Val = 1;
 } // expected-note{{uninitialized use occurs here}}
 
+void Use(int V) {}
```

Comment:
Let's also add a case where we call a function with a signature like `void Def(out int V)` to show that there's no warning there.